core/templates/base-components/thanks-for-subscribing-modal.component.ts

```diff
@@ -13,13 +13,12 @@
 // limitations under the License.
 
 /**
- * @fileoverview Controller for the donation page thanks for subscribing modal.
+ * @fileoverview Controller for the thanks for subscribing modal.
```

Comment:
Maybe add hyphens: thanks-for-subscribing

Reply:
Thanks, Done